Typist.swift

```diff
@@ -119,8 +119,10 @@ public class Typist: NSObject {
     
     internal func keyboardOptions(fromNotificationDictionary userInfo: [AnyHashable : Any]?) -> KeyboardOptions {
         var currentApp = false
```

Comment:
I think this line should be changed to `true` since keyboard always belong to current app in iOS 8.